assets/js/src/navigation.js

```diff
@@ -38,9 +38,9 @@ function initNavToggleSubmenus() {
 		return;
 	}
 
-	navTOGGLE.forEach( function( nav ) {
-		initEachNavToggleSubmenu( nav );
-	});
+	for ( let i = 0; i < navTOGGLE.length; i++ ) {
+		initEachNavToggleSubmenu( navTOGGLE[ i ]);
```

Comment:
Can you fix the spacing to be `initEachNavToggleSubmenu( navTOGGLE[i] );`?

Reply:
Done, but the build script reports a syntax error (`42:42  error  There should be no spaces inside this paren  space-in-parens`). Any idea why? Cc @ataylorme.